Automattic/jetpack
# VideoPress plugin: Hijack the edit attachment page

#### Changes proposed in this Pull Request:

In the VideoPress plugin, we will have a central place to edit video details. This will be the one source of truth for video metadata.

This PR disables the editing of metadata from the Media library, that currently only works in the Jetpack plugin, and hijacks the link to the edit page of the video (this page is not implemented yet)

This will still need an iteration of design in a following PR.

#### Other information:


#### Jetpack product discussion


pe4Cmx-40-p2#comment-13

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Make sure you have at least a couple of attachments and one VideoPress video uploaded to your site
* Activate VideoPress plugin
* Go to Media
* In your Media library, browse your attachments
* Make sure attachments that are not videopress video still work as usual
* Make sure the video details on the right does not give you the option to edit and show only info like the image below
![Captura de tela de 2022-08-22 15-50-03](https://user-images.githubusercontent.com/971483/185997180-b3df3cdc-1fb1-4dbf-9293-4a4b7123589a.png)

* Make sure the "Edit video details" links to the plugins page, with a `#/video/$ID/edit` route in the URL

* Create a new post
* Add a VideoPress block
* Click "Media Library"
* click on a VideoPress video
* Make sure the video details on the right does not give you the option to edit and show only info like the image below
![Captura de tela de 2022-08-22 15-49-54](https://user-images.githubusercontent.com/971483/185997364-e73c5fdc-a426-4da9-8aae-56c1a389a5d3.png)

* Make sure the "Edit video details" links to the plugins page, with a `#/video/$ID/edit` route in the URL



---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - 0-as-1202813985335453
